src/bin/jcli_app/debug/message.rs

```diff
@@ -0,0 +1,27 @@
+use cardano::util::hex;
+use chain_core::property::Deserialize as _;
+use chain_impl_mockchain::message::Message as MockMessage;
+use std::fs::File;
+use std::io::{self, Read};
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub struct Message {
+    /// file containing hex-encoded message. If not provided, it will be read from stdin.
+    #[structopt(short, long)]
+    input: Option<PathBuf>,
+}
+
+impl Message {
+    pub fn exec(self) {
+        let mut hex = String::new();
+        match self.input {
+            Some(path) => File::open(path).unwrap().read_to_string(&mut hex).unwrap(),
+            None => io::stdin().read_line(&mut hex).unwrap(),
+        };
```

Comment:
There is a function in `jcli_app::utils::io` to help you read from the input or the given file